### DISCUSSION TITLE
What does "Number of x-y coordinates in `mesh` does not match `nrow(data)`." indicate?

### DISCUSSION THREAD

**Discussion by StephLingard** [sdmTMB/sdmTMB#137] [Q&A] [answered]
Hello,

Attempting to use this package for the first time. Have made mesh with my data and tried to fit spatial model but get error "Number of x-y coordinates in `mesh` does not match `nrow(data)`." 

My dataset is 20 specific sites fished bi-weekly for 6 months (exact replications of points every two weeks). Could this be causing an issue? Or is it likely that I'm not specifying the number of knots or cutoff values in the mesh correctly?

Many thanks, 

Stephanie

**Comment by seananderson** [comment, marked as answer]
Most likely you made your mesh via `make_mesh()` with a different data frame from the one going into the `data` argument in `sdmTMB()`. Could that be the case?

**Comment by StephLingard** [comment]
Thank you!
